### DISCUSSION TITLE
GitHub and Unreal Engine 5 - Landscape Issue when Updating to main branch

### DISCUSSION THREAD

**Discussion by AddamarDan** [community/community#109240] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Show & Tell

### Body

Hello,

I have recently started a project with a couple of friends, and we are using GitHub as our go-to-software to get assets, progress from eachother, etc.

There is one problem though, if me and my friends are editing the same Landscape (either sculpt it, smoothen it, paint it, basically everything that is altering the landscape shape/color), we will have conflicts when we commit and the changes are not merging but instead, making the edited landscape (by both sides) disappear completely when we update from “main branch”.

Can anyone help us figure this out? It’s ruining our project and we are stuck with this issue.

Thank you very much for your help in advance!

Here is a pic how the project looks after we both edited the landscape (me from my end, and my friend from the other, then commiting on github and updating from the “main branch”, instead of the landscape merging together, it completely disappeared from both ends)
![2024-02-19 21_15_17-Window](https://github.com/community/community/assets/145563987/cbec8bd3-a797-4827-8871-6a20d645b3b9)

**Comment by ReuAzel181** [comment]
Firstly, communicate and coordinate with your team before making significant changes to a landscape to avoid conflicts. Coordinate your efforts to minimize overlapping edits. Secondly, use branches instead of editing the landscape directly on the main branch. Create separate branches for each team member to work on, allowing for independent work without affecting others. Thirdly, when merging branches, use Git's merge tools to resolve conflicts manually. This way, you can review and select which changes to keep, ensuring that the landscape is not lost. Additionally, consider using a version control system that supports locking or versioning for binary files to prevent conflicts and data loss. Lastly, back up your project regularly to have the option to restore previous versions if conflicts arise.
